src/Driver/SQLite3/Statement.php

```diff
@@ -40,7 +40,13 @@ public function __construct(SQLite3 $connection, SQLite3Stmt $statement)
         $this->statement  = $statement;
     }
 
-    /** @inheritdoc */
+    /**
+     * @throws UnknownParameterType
+     *
+     * @inheritdoc
```

Comment:
```suggestion
     * {@inheritdoc}
```

Reply:
I could make a different PR updating this in the whole repo if you agree. There are much more occurrences.

Reply:
It was correct when it was on its own, so it's fine when you're doing it only for the two occurrences of this PR.